learning_resources/management/commands/backpopulate_favorites_lists.py

```diff
@@ -0,0 +1,44 @@
+"""Management command to create user Favorites lists"""
+from django.contrib.auth.models import User
+from django.core.management import BaseCommand
+
+from learning_resources.constants import FAVORITES_TITLE
+from learning_resources.models import UserList
+from learning_resources.plugins import AuthenticationHooksPlugin
+from open_discussions.utils import now_in_utc
+
+
+class Command(BaseCommand):
+    """Create a Favorites userlist for every active user"""
+
+    help = "Create a Favorites userlist for every active user"  # noqa: A003
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--delete",
+            dest="delete",
+            action="store_true",
+            help="Delete all existing Favorites user lists",
+        )
+        super().add_arguments(parser)
+
+    def handle(self, *args, **options):  # noqa: ARG002
+        """Create a Favorites userlist for every active user"""
+        if options["delete"]:
+            self.stdout.write("Deleting all existing Favorites userlists")
+            UserList.objects.filter(title=FAVORITES_TITLE).delete()
+        else:
+            self.stdout.write("Creating Favorites lists for each user")
+            start = now_in_utc()
+            user_ids = UserList.objects.filter(title=FAVORITES_TITLE).values_list(
+                "author_id", flat=True
+            )
+            plugin = AuthenticationHooksPlugin()
+            for user in User.objects.exclude(id__in=user_ids).exclude(is_active=False):
+                plugin.create_favorites_list(user)
```

Comment:
This code is broken not because the imported class doesn't exist, and the function is `user_created` - I don't think it makes sense to call the plugin anymore and it should either just do the `get_or_create` directly - we'll only use this command once anyway.